src/test_helpers/component_tests_preload.cljs

```diff
@@ -1,9 +1,11 @@
 (ns test-helpers.component-tests-preload
   {:dev/always true}
   (:require
-    [reagent.core :as reagent]
     ;; NOTE: Do NOT sort i18n-resources because it MUST be loaded first.
     [status-im.setup.i18n-resources :as i18n-resources]
+   ;; NOTE: reagent.config Must be before any component definition,
```

Comment:
I suggest adding `#_{:clj-kondo/ignore [:unsorted-required-namespaces]}` before `[reagent.config]`. Btw, you probably know but maybe prefer this way (?) but the square brackets are unnecessary here and could be just `reagent.config`.

Reply:
Yes, I prefer it in that way because of consistency with other namespaces having `[ ]`.

But it's fine, I removed the square brackets :+1:  I don't have a strong opinion on it.

Reply:
I'm also like you and would prefer square brackets were always used.